test/function/TEST_JSON_COLUMN_FUNCTION.sql

```diff
@@ -0,0 +1,177 @@
+-- TEST_DELETE_LOGS.sql
+--
+-- Author:      Felix Kunde <felix-kunde@gmx.de>
+--
+--              This script is free software under the LGPL Version 3
+--              See the GNU Lesser General Public License at
+--              http://www.gnu.org/copyleft/lgpl.html
+--              for more details.
+-------------------------------------------------------------------------------
+-- About:
+-- Script that checks whether the json composing util works correctly
+-------------------------------------------------------------------------------
+--
+-- ChangeLog:
+--
+-- Version | Date       | Description                                    | Author
+-- 0.7.3     2022-09-12   initial commit                                   FKun
+--
+
+-- get test number
+SELECT nextval('pgmemento.test_seq') AS n \gset
+
+\echo
+\echo 'TEST ':n': pgMemento test util functions'
+
+\echo
+\echo 'TEST ':n'.1: pgmemento.column_array_to_column_list should work with 1 column'
+DO
+$$
+DECLARE
+  result TEXT;
+BEGIN
+  SELECT
+    pgmemento.column_array_to_column_list(ARRAY['col'])
+  INTO
+    result;
+
+  ASSERT result = 'jsonb_build_object(''col'', col)', 'Error: Columns not concatenated correctly';
+END;
+$$
+LANGUAGE plpgsql;
+
+\echo
+\echo 'TEST ':n'.1: pgmemento.column_array_to_column_list should work with 2 columns'
+DO
+$$
+DECLARE
+  result TEXT;
+BEGIN
+  SELECT
+    pgmemento.column_array_to_column_list(ARRAY['col', 'other_column'])
+  INTO
+    result;
+
+  ASSERT result = 'jsonb_build_object(''col'', col, ''other_column'', other_column)', 'Error: Columns not concatenated correctly';
+END;
+$$
+LANGUAGE plpgsql;
+
+\echo
+\echo 'TEST ':n'.1: pgmemento.column_array_to_column_list should work with 100 columns'
+DO
+$$
+DECLARE
+  result TEXT;
+BEGIN
+  SELECT
+    pgmemento.column_array_to_column_list(ARRAY[
+      'col1',
```

Comment:
can you use `generate_series` to create an array of this size. For the expected result it's fine. Question is if all the tests are still needed when we stop using `jsonb_build_object`. On small test with expected string would be enough, I guess.

Reply:
I just got rid of this in general since the new logic is not really in any way touching the 100 column limit.